components/static.html

```diff
@@ -29,11 +29,36 @@
   }
 
   .slowfade {
-      -webkit-transition: opacity 1.5s; /* For Safari 3.1 to 6.0 */
-      transition: opacity 1.5s;
-      opacity: 0;
+    -webkit-transition: opacity 1.5s; /* For Safari 3.1 to 6.0 */
+    transition: opacity 1.5s;
+    opacity: 0;
+  }
+
+  #ieAlert {
+    display: none;
   }
 </style>
+<!--Bad browser error message-->
+<div id="ieAlert" ng-hide="portal.hideIEAlert" class="md-dialog-container" style="top: 0px; height: 100%; width: 100%; background: rgba(0,0,0,0.3); z-index: 2001;">
+  <md-dialog style="background: #fff;" class="md-transition-in" role="alertdialog" tabindex="-1">
+    <md-dialog-content class="md-dialog-content" role="document" tabindex="-1">
+      <h2 class="md-title">MyUW works best in Firefox and Chrome</h2>
+      <div class="md-dialog-content-body">
+        <a href="https://kb.wisc.edu/page.php?id=44398" target="_blank" style="color: #0479a8; text-decoration: none; font-size: 14px; line-height: 20px;">Get help updating your browser.</a>
```

Comment:
Consider linking more-specific-to-recommended-browsers-for-MyUW https://kb.wisc.edu/page.php?id=51345